saas/static/js/djaodjin-saas-vue.js

```diff
@@ -2156,7 +2107,6 @@ Vue.component('plan-subscriber-list', {
                             email: str,
                             full_name: str
                         }
-                        vm.modalShow();
```

Comment:
Why did this statement get removed from the `plan-subscriber-list` component (which is used in both `templates/saas/profile/plans/subscribers.html` and `templates/saas/profile/subscribers.html`) and wasn't replaced by something like `vm.formVisible = true;` as done previously?

Reply:
there is no such method in this component or any other mixin, so it was likely throwing exceptions anyway